src/views/Error.vue

```diff
@@ -22,15 +25,24 @@
       <!-- failsafe begins -->
       <div v-if="hasFailSafe">
 
+        <!-- TODO - Adding loading spinner till YT video loads -->
         <!-- failsafe G-form begins -->
-        <div v-if="isFailSafeGform">
+        <div v-if="isFailSafeGform && isVideoIdFetched" >
```

Comment:
Added a computed property - `isVideoIdFetched`

Reply:
Referring to the other comment, fetching probably not needed